codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -805,6 +796,29 @@ private InterfaceHttpData findMultipartDisposition() {
         }
     }
 
+    private static final Pattern DOUBLE_SINGLE_QUOTE = Pattern.compile("''");
+    private static final String FILENAME_ENCODED = HttpHeaderValues.FILENAME.toString() + '*';
+
+    private Attribute getContentDispositionAttribute(String... values) {
+        String name = cleanString(values[0]);
+        String value = values[1];
+
+        // See http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html
+        if (HttpHeaderValues.FILENAME.contentEquals(name)) {
+            // filename value is quoted string so strip them
+            value = value.substring(1, value.length() - 1);
```

Comment:
consider checking that the first and last chars are actually quotes.

Reply:
this was here before this PR. should i modify this behavior in this PR?

Reply:
Adding this check too.